openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -128,8 +128,8 @@ def update_op_assets(
             if frames_duration:
                 frame_out = frame_in + frames_duration - 1
             else:
-                frame_out = project_doc["data"].get("frameEnd", frame_in)
-        item_data["frameEnd"] = frame_out
+                frame_out = int(project_doc["data"].get("frameEnd", frame_in))
+        item_data["frameEnd"] = int(frame_out)
```

Comment:
No need to cast it twice
```suggestion
                frame_out = project_doc["data"].get("frameEnd", frame_in)
        item_data["frameEnd"] = int(frame_out)
```